python/cog/server/worker.py

```diff
@@ -288,20 +288,14 @@ def run(self) -> None:
         # We use SIGUSR1 to signal an interrupt for cancelation.
         signal.signal(signal.SIGUSR1, self._signal_handler)
 
-        ws_stdout = WrappedStream("stdout", sys.stdout)
-        ws_stderr = WrappedStream("stderr", sys.stderr)
-        ws_stdout.wrap()
-        ws_stderr.wrap()
-
         self._stream_redirector = StreamRedirector(
-            [ws_stdout, ws_stderr], self._stream_write_hook
+            tee=self._tee_output,
```

Comment:
Had a look at this and it requires some bigger changes (because `_ChildWorker` is a `Process` subclass) that I don't particularly want to couple to this change.

Reply:
Fair enough.